generated/.tailcallrc.graphql

```diff
@@ -633,6 +634,13 @@ enum LinkType {
   Cert
   Key
 }
+input Lint {
+  autoFix: Boolean!
+  enumName: Boolean!
```

Comment:
How can we approaches for requested changes in #1330 .

@coderabbitai

Reply:
please explain in detail what is requested?
@coderabbitai

Reply:
What @1330 issue requests:
Current
The current lint configuration allows the following:

input Lint {
  autoFix: Boolean
  enum: Boolean
  enumValue: Boolean
  field: Boolean
  type: Boolean
}
Proposed
We should provide a more flexible way to configure the schema.

input Lint {
  autoFix: Boolean
  enum: TextCase
  enumValue: TextCase
  field: TextCase
  type: TextCase
}

enum TextCase {
  CAMEL_CASE
  PASCAL_CASE
  SNAKE_CASE
  SCREAMING_SNAKE_CASE
}


Here is the requested changes,
Please explain in clarity.
example use case.

Reply:
@coderabbitai
look at the above comment.

Reply:
So, we need to make case validator for each enum value type.
@coderabbitai

Reply:
So, we need to make case validator for each enum value.
thanks.
@coderabbitai

Reply:
explain  again.
@coderabbitai

Reply:
So, we need to make case validator for each enum value.
Explain on this more.
@coderabbitai

Reply:
@example
@coderabbitai

Reply:
example
@coderabbitai

Reply:
> @webbdays, assuming you're looking for an example related to our discussion on implementing case validators for enum values in the GraphQL schema, here's a conceptual example:
> 
> ### Example: Implementing a Case Validator for Enum Values
> Let's say you want to ensure that all enum values in your GraphQL schema adhere to `SCREAMING_SNAKE_CASE`. You could implement a simple case validator function in JavaScript:
> 
> ```js
> function isScreamingSnakeCase(str) {
>   return /^[A-Z0-9_]+$/.test(str);
> }
> 
> // Example enum value
> const enumValue = "THIS_IS_AN_EXAMPLE";
> 
> // Validate the enum value
> if (isScreamingSnakeCase(enumValue)) {
>   console.log("The enum value adheres to SCREAMING_SNAKE_CASE.");
> } else {
>   console.log("The enum value does not adhere to SCREAMING_SNAKE_CASE.");
> }
> ```
> 
> This function checks if a given string (`str`) matches the `SCREAMING_SNAKE_CASE` convention using a regular expression. You could integrate such a function into your linting process to validate enum values in your GraphQL schema.
> 
> If you were seeking an example related to a different topic or have specific requirements, please provide more details so I can assist you accordingly.

Where to call such fn.
@coderabbitai